kong/kong.lua

```diff
@@ -144,9 +144,11 @@ function Kong.init()
 end
 
 function Kong.init_worker()
+  prepare_module_variables()
```

Comment:
s/globals/upvalues, since we're dealing with a module required (thus, loaded in `package.preload`), the values should follow when the Lua VM is forked in the workers.